python_src/src/lamp_py/ingestion/config_rt_vehicle.py

```diff
@@ -20,15 +20,39 @@ def import_schema(self) -> pyarrow.schema:
                     "vehicle",
                     pyarrow.struct(
                         [
-                            ("current_status", pyarrow.string()),
+                            ("trip", trip_descriptor),
+                            ("vehicle", vehicle_descriptor),
+                            ("position", position),
                             ("current_stop_sequence", pyarrow.uint32()),
-                            ("occupancy_percentage", pyarrow.uint32()),
-                            ("occupancy_status", pyarrow.string()),
                             ("stop_id", pyarrow.string()),
+                            ("current_status", pyarrow.string()),
                             ("timestamp", pyarrow.uint64()),
-                            ("position", position),
-                            ("trip", trip_descriptor),
-                            ("vehicle", vehicle_descriptor),
+                            ("congestion_level", pyarrow.string()),
+                            ("occupancy_status", pyarrow.string()),
+                            ("occupancy_percentage", pyarrow.uint32()),
+                            (
+                                "multi_carriage_details",
+                                pyarrow.list_(
+                                    pyarrow.struct(
+                                        [
+                                            ("id", pyarrow.string()),
+                                            ("label", pyarrow.string()),
+                                            (
+                                                "occupancy_status",
+                                                pyarrow.string(),
+                                            ),
+                                            (
+                                                "occupancy_percentage",
+                                                pyarrow.int32(),
+                                            ),
+                                            (
+                                                "carriage_sequence",
+                                                pyarrow.uint32(),
+                                            ),
+                                        ]
+                                    )
+                                ),
+                            ),
```

Comment:
this comes from the spec you saw this morning?  seems weird that we have this info in the schema twice?

Reply:
This is on a per car basis, where the other values (one level up) would represent the entire series of cars